platform-ui-skin/pom.xml

```diff
@@ -37,6 +37,7 @@
             <plugin>
                 <groupId>org.lesscss</groupId>
                 <artifactId>lesscss-maven-plugin</artifactId>
+                <version>1.7.0.1.1</version>

```

Comment:
I'm just working on a new skin via the platform-ui project, so I didn't want to go changing things that weren't (strictly) part of the project.

I have submitted an updated pull request :)